Automattic/wp-calypso
# Importer: Revert to Business Plan

#### Proposed Changes

See https://github.com/Automattic/wp-calypso/pull/65784 and pdgrnI-1sK-p2#comment-2605 ns for revert.

#### Testing Instructions

Make sure both the Tools > Import workflow and the onboarding import workflow show the old Business plan.

<img width="767" alt="CleanShot 2022-07-21 at 11 37 03@2x" src="https://user-images.githubusercontent.com/533/180142567-cf73ba76-7d3c-479d-af9d-414a8419dd5e.png">
<img width="961" alt="CleanShot 2022-07-21 at 11 37 58@2x" src="https://user-images.githubusercontent.com/533/180142554-a6aad07b-df04-453a-8758-ec70b32d149b.png">
